internal/transport/transport_test.go

```diff
@@ -461,11 +461,15 @@ func setUpWithOptions(t *testing.T, port int, sc *ServerConfig, ht hType, copts
 }
 
 func setUpWithNoPingServer(t *testing.T, copts ConnectOptions, connCh chan net.Conn) (*http2Client, func()) {
+	return setUpWithNoPingServerWithOptions(t, copts, connCh, nil)
+}
+
+func setUpWithNoPingServerWithOptions(t *testing.T, copts ConnectOptions, connCh chan net.Conn, switchPingChan chan bool) (*http2Client, func()) {
```

Comment:
Please add a comment to the helper. This would be help make more sense of what `WithOptions` means her.